src/components/error-message/index.md.njk

```diff
@@ -1,6 +1,6 @@
 ---
 title: Error message
-description: Use an error message when there is a validation error. Explain what went wrong and how to fix it
+description: When there's a validation error, use an error message to explain what went wrong and how to fix it
```

Comment:
```suggestion
description: When there’s a validation error, use an error message to explain what went wrong and how to fix it
```

Reply:
The Markdown parser we're using takes care of quotes automatically, converting them to the appropriate character.